docs/user-guide/at-tls-configuration.md

```diff
@@ -1,23 +1,642 @@
+}
+```
+
+:::important
+
+- Outbound connections from the Gateway to southbound services (onboarded services) must not send the server certificate if the service accepts X.509 Client Certificate authentication. If the server certificate is sent, it is the server user who would be authenticated.
+
+:::
+
+### Ciphers
+
+:::note
+This list of ciphers is provided as an example only. Actual ciphers should be customized according to your specific configuration.
+:::
+
+The list of supported ciphers should be constructed according to the TLS supported versions.
+Ensure that the cipher list has matches with non-AT-TLS-aware clients.
+
+<details>
+<summary>Click here for an example of Cipher parameters.</summary>
+
+```bash
+TTLSCipherParms CipherParms
+{
+  V2CipherSuites TLS_RC4_128_WITH_MD5
+  V2CipherSuites TLS_RC4_128_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_DES_192_EDE3_CBC_WITH_MD5
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_AES_128_GCM_SHA256
+  V3CipherSuites TLS_AES_256_GCM_SHA384
+  V3CipherSuites TLS_CHACHA20_POLY1305_SHA256
+}
+```
+
+</details>
+
+## Using AT-TLS for API ML in High Availability
+
+AT-TLS settings for a Zowe installation configured in High Availability mode do not differ extensively. Changes need to be made to the previously described rules to allow for cross-lpar communication:
+
+Ensure that the `RemoteAddr` setting in the rules accounts for the following connections:
+
+- Discovery Service to Discovery Service. This is the replica request.
+- Gateway Service to southbound services (including app-server and ZSS) running in another LPAR.
+- Southbound services to Discovery Service. This applies during onboarding.
+
+## Multi-tenancy deployment
+
+For specific scenario when Central API ML is running on z/OS with AT-TLS enabled, it is important to override the protocol for the external URL. This information is used by the Central API ML to call domain API ML and it needs to reflect outbound AT-TLS rule. In this case, update your domain API ML configuration as follows:
+
+```yaml
+zowe:
+  components:
+    gateway:
+      apiml:
+        gateway:
+          externalProtocol: http
+```
+
+## AT-TLS Troubleshooting
+
+This section describes some common issues when using AT-TLS with Zowe and how to resolve these issues.
+
+### The message `This combination of port requires SSL` is thrown
+
+Make sure the URL starts with `https://`. This message indicates that AT-TLS rules are in place and it is trying to connect on port 80 to the API Gateway, however the latter is still only listening on the secure port 443.
```

Comment:
I guess we should rather describe the issue and avoid ports, because ports are not ussually 80/443

```suggestion
It could happen if the connection tries to be twice encoded (by service and by AT-TLS). Make sure the URL starts with `http://` or AT-TLS user is applied. Never combine both configurations.
```